src/menus/FileMenus.cpp

```diff
@@ -553,12 +549,16 @@ auto FileMenu()
          )
       ),
 
+      Section( "Close", Command( wxT("Close"), XXO("&Close Project"), OnClose,
```

Comment:
you don't need the `Section("Close"` bit anymore, you can instead put the `Command(...)` thing into the Exit section.